src/dataframe/sort.jl

```diff
@@ -12,7 +12,14 @@ end
 function Base.sort!(df::DataFrame, a::Base.Sort.Algorithm, o::Base.Sort.Ordering)
     p = sortperm(df, a, o)
     pp = similar(p)
-    for col in columns(df)
+    c = columns(df)
+
+    for (i,col) in enumerate(c)
+        # Check if this column has been sorted already. PR#1072
```

Comment:
No PR names in the code, `git blame` is there for this. (I know, it's a bit confusing that we keep them in tests, but...)